core/client/mutations.go

```diff
@@ -39,16 +39,13 @@ type RevisionMutations struct {
 // StreamRevisions repeatedly fetches revisions and sends them to out until GetRevision
 // returns an error other than NotFound or until ctx.Done is closed.  When
 // GetRevision returns NotFound, it waits one pollPeriod before trying again.
-func (c *Client) StreamRevisions(ctx context.Context, directoryID string, startRevision int64, out chan<- *pb.Revision) error {
+func (c *Client) StreamRevisions(ctx context.Context, startRevision int64, out chan<- *types.MapRootV1) error {
 	defer close(out)
 	wait := time.NewTicker(c.RetryDelay).C
 	for i := startRevision; ; {
 		// time out if we exceed the poll period:
```

Comment:
Can you format this comment?

Also, is it accurate? It's not obvious from the `VerifiedGetRevision` invocation that it has some timing out built in.

Reply:
It's not accurate